apps/sms/index.html

```diff
@@ -25,12 +25,14 @@
     <link rel="stylesheet" type="text/css" href="views/shared/style/edit-mode.css">
     <link rel="stylesheet" type="text/css" href="views/shared/style/notification.css">
     <link rel="stylesheet" type="text/css" href="views/shared/style/common.css">
+    <link rel="stylesheet" type="text/css" href="views/shared/style/composer.css">
     <link rel="stylesheet" type="text/css" href="views/conversation/style/message.css">
     <link rel="stylesheet" type="text/css" href="views/conversation/style/report_view.css">
     <link rel="stylesheet" type="text/css" href="views/conversation/style/attachment.css">
-    <link rel="stylesheet" type="text/css" href="views/conversation/style/composer.css">
     <link rel="stylesheet" type="text/css" href="views/conversation/style/conversation.css">
     <link rel="stylesheet" type="text/css" href="views/conversation/style/edit-mode.css">
+    <link rel="stylesheet" type="text/css" href="views/new_message/style/new_message_view.css">
```

Comment:
nit: do you think we need ```_view``` postfix in ```new_message_view.css```? I haven't added it for inbox and conversation, so for the consistency sake we can either remove ```_view``` here (I'd rather go this way :)) or add ```_view``` to other files later, wdyt?